tests/benchmark/performance/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# Tests need to be added as executables first
+file(GLOB BENCHMARK_PERFORMANCE_TEST_LIST CONFIGURE_DEPENDS
```

Comment:
i belive GLOB is not the preferred way in cmake

https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak